sandbox.html

```diff
@@ -406,6 +408,15 @@
   </div>
 </template>
 
+<template data-action="setSelection">
+  <div>
+    <label for="layerPath">Path to the dataview layer * (GeoJSON Array)</label>
```

Comment:
* I think it should be "data view layer" to be proper English.
* It's not a GeoJSON array.